packages/react-components/src/components/VideoTile.tsx

```diff
@@ -351,13 +355,15 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
     hidePersonaDetails: true
   };
 
-  const videoHintWithBorderRadius = mergeStyles(videoHint, { borderRadius: theme.effects.roundedCorner4 });
+  const videoHintWithBorderRadius = mergeStyles(disabledVideoHint, {
+    borderRadius: theme.effects.roundedCorner4,
+    backgroundColor: isDarkThemed(theme) ? 'rgba(37,36,35,0.8)' : 'rgba(255,255,255,0.8)'
```

Comment:
Do we want to instead of hard coding them assign them to the palettes we provide so that contoso can set them in the theme too?

Reply:
Add a new theme call videoTileLabelBackgroundLight.
Great suggestion!